doc/README.netcdf4par

```diff
@@ -12,6 +12,9 @@ setenv NETCDFPAR /usr/local/netcdf474par
 (This will also cause configure to set NETCDF = NETCDFPAR to prevent 
 conflicting libraries, and also will force NETCDF4=1 and USENETCDFPAR=1)
 
+To use non-parallel netcdf-4, set USENETCDFPAR=0 in the configuring
+environment, as the default assumption is that netcdf-4 is parallel.
```

Comment:
This may only be necessary when on a system without MPI; should I focus on that instead?

Reply:
```suggestion
To use netcdf-4 in an environment without MPI, set USENETCDFPAR=0 in
the configuring environment, as the default assumption is that the
netcdf-4 check requires MPI to compile.
```